tutorials/iot-onboard-device/iot-onboard-device.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Onboard Virtual Devices sending Data from your Computer
+description: Allows you to to register new devices in SAP IoT to allow it to send data.
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, products>sap-leonardo-iot]
+primary_tag: topic>internet-of-things
+---
+
+## Prerequisites
+ - **Tutorials:** [Create Model for your devices in the DEV tenant](iotae-api-postman)
+
+## Details
+### You will learn
+  - How to quickly onboard multiple devices
+  - How to trigger that devices send mock data regularly
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Decide which devices to onboard)]
+
+Onboarding of devices is not just something that you do one-time when establishing a data model for your IoT application. You do this continuously (and also off-boarding) during the operation of your IoT application. To make this a smooth process where you can be sure the device works but also all other systems are aware fo the device automating this with scripts is essential. We will also use a script in this tutorial.
+
+When onboarding a device and when doing this in the context of a business application you need to bring together the technical activation and the application integration into a coherent process. For example a sim card might need to be activated or a handling unit in ERP might need to know, that there is now a tracker which is supposed to send data on the geographic position of the handling unit.
+
+So to be able to do this one first has to collect the technical ids of the involved devices and the application ids of the involved application objects as a basis. Such a list could look like this:
+
+```
+[
+  {"id":"EQUI0001_01","EQUI":"EQUI0001_01","HANU":"HANU5658","IMEI":"0456456580934"},
+  {"id":"EQUI0001_02","EQUI":"EQUI0001_02","HANU":"HANU4234","IMEI":"9823065469902"},
+  {"id":"EQUI0001_03","EQUI":"EQUI0001_03","HANU":"HANU7458","IMEI":"0983409580934"},
+  {"id":"EQUI0001_04","EQUI":"EQUI0001_04","HANU":"HANU4568","IMEI":"9823098409902"}
+]
+```
+
+In above case we have 2 process application identifiers, one for equipment maintenance and one for handling units used during sales order processing. And then there is a technical id that refers to a telemetry module's IMEI number (check your mobile phone system properties for a similar IMEI number). Both the IMEI and the EQUI number could be used as the device id as both will be assigned long-term to the device - which one you use really depends on what the device is capable of using. Let's assume for now the device "knows" its equipment id. The HANU id is subject to change or might be unknown during onboarding as the silos, we are considering in this example, could be used as a handling unit with one id on Monday and get a different one on Tuesday assigned dynamically. So this would not be a good device id.
```

Comment:
```suggestion
In above case we have 2 process application identifiers, one for equipment maintenance and one for handling units used during sales order processing. And then there is a technical id that refers to a telemetry module's IMEI number (check your mobile phone system properties for a similar IMEI number). Both the IMEI and the EQUI number could be used as the device id as both will be assigned to the device on a long-term basis- which one you use really depends on what the device is capable of using. Let's assume for now the device "knows" its equipment id. The HANU id is subject to change or might be unknown during onboarding as the silos, we are considering in this example, could be used as a handling unit with one id on Monday and get a different one on Tuesday assigned dynamically. So this would not be a good device id.
```

Reply:
i edited this to have less ands